tbx/project_styleguide/templates/patterns/pages/team/team_detail.yaml

```diff
@@ -47,8 +47,14 @@ tags:
     '':
       template_name: 'patterns/navigation/components/footer-links.html'
   pageurl:
-    work_page.specific.work_index as work_index_url:
+    post.get_parent as blog_index_url:
+      raw: '../blog/blog_listing.html'
```

Comment:
I like this - presumably it means it links to the listing page in the pattern library in this case?

Reply:
That is correct. I thought "what happens if we do that" it was a nice surprise. Works even better when you preview the template separately (or full-screen)